wpeverest/everest-forms
# Fix - Background color for PDF submission

### All Submissions:






### Changes proposed in this Pull Request:




This PR addresses the issue where the background color for PDF submissions was only applied to the body section, leaving the footer section without the intended background color. The update ensures that the background color is consistently applied across the entire PDF, including both the body and footer sections, improving the visual consistency of the submissions.

Closes # .

### How to test the changes in this Pull Request:

1. Enable **Everest Forms PDF Submission addon**
2. Create a Form or you can use the existing form
3. Now go to **Everest Forms Settings > PDF Submission**
4. Select **Background Color** and click on **Save Button**
5. Now you can either download the form after submission or Got to **Entries > Export to PDF** to check this.

[PR Dependency](https://github.com/wpeverest/everest-forms-pro/pull/972)

### Types of changes:




### Other information:




### Changelog entry

> Fix - Background color for PDF submission.
